hls4ml/writer/quartus_writer.py

```diff
@@ -596,11 +621,33 @@ def __write_exp_table(self, model, path):
 
         h_file = open('{}/{}.tb'.format(path, table_name), 'w')
         h_file.write(self.__get_table_header(table_name, table_size))
-        
+
+        # Default fixed point precision
+        # 6 bits for integer part, 10 bits for decimal - total, 16
+        fp_bits = 16
+        fp_integer = 6
+        fp_signed = True
+
+        # Exp table should use the same precision as exp_table, as seen in Vivado code
+        # init_exp_table<data_T, CONFIG_T>(exp_table);
+        for layer in model.get_layers():
+            if layer.name == 'softmax':
+                ac_type = layer.get_attr('accum_t')
```

Comment:
I couldn't find an attribute for `input_t` in the softmax layer, only manage to find `result_t` and `accum_t`. Is `accum_t` equivalent to `input_t` in this case?

Reply:
Sorted, changed to input